monai/metrics/frd.py

```
@@ -0,0 +1,71 @@
+        y_pred: Feature vectors for the first distribution, shape (N, F).
+        y: Feature vectors for the second distribution, shape (N, F).
+
+    Returns:
+        Scalar tensor containing the Fréchet Radiomics Distance.
+
+    Raises:
+        ValueError: When either tensor has more than 2 dimensions. Inputs must have
+            shape (number of samples, number of features).
+    """
+    if y_pred.ndimension() > 2 or y.ndimension() > 2:
+        raise ValueError("Inputs should have (number images, number of features) shape.")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)